packages/KModal/KModal.vue

```diff
@@ -182,47 +272,101 @@ export default {
   overflow: auto;
 }
 
-.k-modal-dialog {
-  position: relative;
-  width: auto;
-  max-width: var(--KModalMaxWidth, 500px);
-  margin: 50px auto;
-  padding: var(--spacing-xl, spacing(xl));
-  border-radius: 4px;
-  border: var(--KModalBorder);
-  box-shadow: 0px 0px 12px 0px var(--black-10, color(black-10));
-  background: #fff;
-  z-index: 9999;
-}
+.k-modal {
+  // use a var to ensure correct sizing of .k-modal-header-image
+  --KModalPadding: var(--spacing-xl, spacing(xl));
 
-.k-modal-content {
-  position: relative;
-  display: flex;
-  flex-direction: column;
+  .k-modal-dialog {
```

Comment:
Not sure where this belongs, but the layout here looks different than on the beta branch
![image](https://user-images.githubusercontent.com/2229946/180491673-9a071930-cf7e-4df9-adad-d46f8968b112.png)

Reply:
This is from theme styles: 
![image](https://user-images.githubusercontent.com/67973710/180494737-3ddb3b67-dd16-4fb5-bdae-ef789aa8cff1.png)